packages/sandpack-core/src/runner/eval.ts

```diff
@@ -40,6 +40,11 @@ export default function (
     delete allGlobals.global;
   }
 
+  // This is a hack to make core-js work...
+  if (window.process.versions) {
```

Comment:
This is a pretty big hack, see the PR description for better (bigger) solutions/refactors